docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+                categories_allowed:
+                    - Holidays
+                    - News
+            - tags:
+                tags_file: blog/tags.md
+
+        nav:
+            - Home: index.md
+            - Install: install.md
+            - Usage: usage.md
+            - Blog:
+                - blog/index.md
+                Tags: blog/tags.md
+        ```
+
+        The tag index will be appended to the configured page, which you should
+        now create at the location specified.
+
+        Note that you can put the tag index page anywhere in your primary
+        navigation, so if you are using tags elsewhere instead of just in your
+        blog then you may want to have the tag index outside the blog section
+        of the navigation.
+
+
+    === "Insider Edition"
+
+        To add a tag index, you add a placeholder in a Markdown file to tell
+        the plugin to insert an index at that point. This means that you
+        can add content before and after the index. Crucially, you can add
+        placeholders in multiple pages, each with a configuration of what
+        subset of tags should be displayed in the index.
+
+        The simplest index page looks like this. Create it under `docs/tags.md`.
+
+        ```markdown
+        # Tag index
+        <!-- material/tags -->
+        ```
+
+        Now, you may want to keep the tags for your blog separate from tags
+        you use in the rest of your page. You can achieve this by assigning
+        the tag index a scope. Put the following under `docs/blog/tags.md`:
+
+        ```markdown
+        # Tag index  for the blog
+        <!-- material/tags { scope: true } -->
+        ```
+
+        You now have two index pages. One that covers the whole site and one
+        that covers only the blog.
+
+        The tags plugin in the Insider Edition is an incredibly powerful tool
+        and we can only scratch the surface of what is possible with it. If you
+        want to explore more after you have worked for this part of the tutorial,
+        have a look at the [tags plugin reference].
+
+[tags plugin reference]: ../plugins/tags.md
+
+## 10. Pagination
+
+Once your blog starts growing, you may not want to pay attention to the number
+of posts displayed per page. By default, the plugin displays up to 10 posts on
+the archive and
+
+
+
+
+## 11. Social Media
+
+The blog plugin works well with the social plugin, which creates social cards
+for all pages, including blog posts. Social cards are images that other systems
+such as social media can display as a preview for content linked to.
+It is easy to get started with the social plugin, true to the motto of
+Material with MkDocs: "batteries included."
+
+!!! example "Add social cards"
+
+    To add social cards to your blog you need to install some dependencies.
+    These differ depending on what operating system you use.
+
+    TODO: paste in installation instructions here - use snippets since this is
```

Comment:
Please don't. This tutorial is already very, very long, IMHO.

Reply:
Perhaps adding social cards to a blog would be a part in the social cards tutorial rather than the other way round?

Reply:
Jup, good idea. A tutorial on how to get up and running, configure and customize social cards could be an excellent link target from the blog target. Something like: oh, you think you're done with what can be done with Material for MkDocs? Well, check those tutorials out! ...